src/main/java/com/sarality/sync/APISyncChangedDataFetcher.java

```diff
@@ -0,0 +1,63 @@
+package com.sarality.sync;
+
+import com.sarality.db.Column;
+import com.sarality.db.Table;
+import com.sarality.db.query.Operator;
+import com.sarality.db.query.Query;
+import com.sarality.db.query.SimpleQueryBuilder;
+
+import java.util.List;
+
+import hirondelle.date4j.DateTime;
+
+/**
+ * Sync data fetcher for all records created or updated after last sync timestamp and upto current sync timestamp
+ *
+ * @author Satya@ (Satya Puniani)
+ */
+
+public class APISyncChangedDataFetcher<T> implements APISyncDataFetcher<T> {
```

Comment:
Let's check this in, but add a TODO that this might need to change based on Locally modified Column